.phive/phars.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<phive xmlns="https://phar.io/phive">
```

Comment:
is this a new file needed for recent versions of humbug/box? or why do we need this file? i don't find any call to a tool "phive" in the MR